src/dmd/target.d

```diff
@@ -95,17 +90,14 @@ struct Target
         }
     }
 
-    ///
-    alias FloatProperties = FPTypeProperties!float;
-    ///
-    alias DoubleProperties = FPTypeProperties!double;
-    ///
-    alias RealProperties = FPTypeProperties!real_t;
+    FPTypeProperties!float FloatProperties;     ///
+    FPTypeProperties!double DoubleProperties;   ///
+    FPTypeProperties!real_t RealProperties;     ///
```

Comment:
Why was a typedef/alias changed to a field ?

Reply:
Because the `__gshared` was removed. See original line 71.

Reply:
Right, I got the nesting wrong. Thanks